.github/workflows/ci.yaml

```diff
@@ -55,13 +55,13 @@ jobs:
     - name: Run unit tests
       if: ${{ !cancelled() }}
       run: |
-        pytest -n 4 .
+        pytest -n 4 --capture=tee-sys -vv .
 
     - name: Run e2e tests on MI300
       if: "contains(matrix.os, 'mi300') && !cancelled()"
       run: |
         export WAVE_RUN_E2E_TESTS=1
-        pytest -n 4 --capture=tee-sys ./tests/kernel/wave/
+        pytest -n 4 --capture=tee-sys -vv ./tests/kernel/wave/
```

Comment:
Should we remove Run e2e tests on MI300 from ci.yaml since it is already in ci-tk.yaml?

Reply:
I'm not sure. My motivation to keep it was that in the bright future, when our CI is not flaky anymore, we can just delete `TK CI` pipeline file and won't need any changes to main CI.

Reply:
I would say we can still do that in the future, but instead of deleting tk ci, we can merge ci and tk ci, though there might be something to be said about compartmentalization of different parts of the codebase. (think changes in one folder only trigger tests related to that) I think since we are trying to save time lets remove as much extraneous work being done on the builders.

Reply:
OK, makes sense. Done.